### DISCUSSION TITLE
Welcome to FlashDB Discussions!

### DISCUSSION THREAD

**Discussion by armink** [armink/FlashDB#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kalpanavengala** [comment]
Hi Sir,
I am using esp-idf 4.2.1 version and Vscode. how to include the flash db library files to make flash with out any errors vscode.
